infra/aws/terraform/prow-build-cluster/vpc.tf

```diff
@@ -21,29 +21,33 @@ limitations under the License.
 # VPC is IPv4/IPv6 Dual-Stack, but our cluster is IPv4 because EKS doesn't
 # support dual-stack yet.
 
+locals {
+  vpc_cidr = "10.0.0.0/12"
+}
```

Comment:
This could still be an input variable, right?

Reply:
It was that way before this, but I thought it might be safer this way. I don't have strong preferences.

Reply:
+1 on having this as a variable.

Reply:
It's an input variable again.